system/jlib/jlog.cpp

```diff
@@ -41,6 +41,16 @@ using namespace ln_uid;
 // Time, in nanoseconds, after which the clock field loops --- 3600000000000ns = 1hr
 #define CLOCK_LOOP_NANOSECONDS I64C(3600000000000)
 
+// Standard filters, handlers, manager, and audit event logger
+static CLogMsgManager * theManager = nullptr;
```

Comment:
Slightly safer to change the type of theManager, and introduce a new pointer for the dynamic manager.  (Missed a few calls e.g. queryJobIDStr(), LogMsgJobInfo::serialize

Reply:
You could alternatively call theManager->Release() and not have two pointers.

Reply:
@ghalliday - I kept both, and left queryJobIDStr() and LogMsgJobInfo::serialize referring to old, because access a member of the class that's not in the interface.

Reply:
I think queryJobIDStr() should really be in the public interface.